library/src/main/res/layout/realm_recycler_view_horizontal.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <android.support.v4.widget.SwipeRefreshLayout
+        android:id="@+id/rrv_swipe_refresh_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/rrv_recycler_view"
+            android:scrollbars="horizontal"
```

Comment:
Looks like `scrollbars` and `layout_height` are the only attributes that are different. These should be settable programmatically and thus eliminate the additional layout.